apps/chatbots/views.py

```diff
@@ -307,9 +311,108 @@ def archive_chatbot(request, team_slug: str, pk: int):
     return HttpResponseClientRedirect(reverse("chatbots:chatbots_home", kwargs={"team_slug": team_slug}))
 
 
-class CreateChatbotVersion(CreateExperimentVersion):
-    permission_required = "experiments.add_experiment"
+class CreateChatbotVersion(LoginAndTeamRequiredMixin, PermissionRequiredMixin, TemplateView):
+    """View for creating a new version of a chatbot."""
+
+    model = Experiment
+    form_class = ExperimentVersionForm
     template_name = "experiments/create_version_form.html"
+    title = "Create Chatbot Version"
+    button_title = "Create"
+    permission_required = "experiments.add_experiment"
+
+    @cached_property
+    def object(self):
+        return get_object_or_404(Experiment, pk=self.kwargs["experiment_id"], team=self.request.team)
+
+    @cached_property
+    def latest_version(self):
+        return self.object.latest_version
+
+    def get_form_kwargs(self) -> dict:
+        form_kwargs = {}
+        if self.request.method == "POST":
+            form_kwargs["data"] = self.request.POST
+        if not self.latest_version:
+            form_kwargs["initial"] = {"is_default_version": True}
+        return form_kwargs
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        form = self.form_class(**self.get_form_kwargs())
```

Comment:
Any reason not to extend `FormView` instead of `TemplateView` and let the parent class handle the form logic?